samples/net/eth_native_posix/README.rst

```diff
@@ -28,15 +28,24 @@ below.
    :compact:
 
 Normally one needs extra privileges to create and configure the TAP device in
-the host system. Use sudo to execute the Zephyr process with admin privileges,
-like this:
+the host system. If the user has set the
+:option:`CONFIG_ETH_NATIVE_POSIX_STARTUP_AUTOMATIC` option (this is disabled
+by default), then the user needs to use ``sudo`` to execute the Zephyr process
+with admin privileges, like this:
 
 .. code-block:: console
 
     sudo --preserve-env=ZEPHYR_BASE make run
 
 If the ``sudo --preserve-env=ZEPHYR_BASE`` gives an error,
-just use ``sudo --preserve-env`` instead:
+just use ``sudo --preserve-env`` instead.
+
+If that option is not enabled (this is the default), then the user should
+execute the ``net-setup.sh`` script from Zephyr `net-tools`_ repository.
+The script should be run before executing the Zephyr process. The script
+will create the zeth interface and setup IP addresses and routes.
+The manual setting does not require the Zephyr process to be run as a root
+user, this is the recommended and preferred way.
```

Comment:
```suggestion
user. This is the recommended and preferred way.
```

Reply:
After this, add some of the details that were in the Kconfig file:

    When the network interface is set up manually, you can leave the wireshark
    to monitor the interface, and then start and stop the zephyr process without
    stopping the wireshark. 

    Setting things manually works the same as working with SLIP
    connectivity in QEMU.